### DISCUSSION TITLE
How to change my term

### DISCUSSION THREAD

**Discussion by Keira-Shuyao** [qu8n/pennplanner#4] [Q&A]
I am a new student of 24 spring, when I log in, I find that there is no 2024 academic year so I choose 23 fall. Then I find the term cannot be reset. Is there anyplace I can change my term or can you add the 2024 academic year in the logging option?

**Comment by qu8n** [comment]
@Keira-Shuyao Thanks for reporting and welcome to the program! The bug has been fixed. Let me know if you run into any other issues.
